services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -51,24 +51,34 @@ function parseSqlDefault(expression) {
   const trues = ['true', 'TRUE', 'b\'1\'', '((1))'];
 
   let result;
-  if (!expression || expression === 'NULL' || expression.startsWith('NULL::')) {
-    result = null;
-  } else if (falses.includes(expression)) {
-    result = false;
-  } else if (trues.includes(expression)) {
-    result = true;
-  } else if (/^'(.*)'::jsonb?$/i.test(expression)) {
-    const [, content] = expression.match(/^'(.*)'::(.*)$/i);
-    result = JSON.parse(content);
-  } else if (/^'(.*)'::[a-z ]+$/i.test(expression)) { // arrays not allowed
-    const [, content] = expression.match(/^'(.*)'::(.*)$/i);
-    result = content;
-  } else if (/^-?\d+(\.\d+)?$/.test(expression)) {
-    result = parseFloat(expression);
-  } else if (/^'.*'$/.test(expression)) {
-    result = expression.substr(1, expression.length - 2);
-  } else {
-    result = Sequelize.literal(expression); // always works
+  try {
+    if (!expression || expression === 'NULL' || expression.startsWith('NULL::')) {
+      result = null;
+    } else if (falses.includes(expression)) {
+      result = false;
+    } else if (trues.includes(expression)) {
+      result = true;
+    } else if (/^-?\d+(\.\d+)?$/.test(expression)) {
+      result = Number.parseFloat(expression);
+      if (result.toString() !== expression) {
+        throw new Error('Bignum overflowing javascript precision');
+      }
+    } else if (/^'.*'$/.test(expression)) {
+      result = expression.substr(1, expression.length - 2);
```

Comment:
Maybe this is missing removing the simple quote escaping.
Need to test what would happen.

Something like`.replace(/\\'/g, "'")`

Reply:
May be to add in a test. I'll try to help you tomorrow on those tests  😉  (If I find some time 😅 )